src/driver.py

```diff
@@ -17,7 +16,7 @@
         config = yaml.safe_load(file)
 # collect data
     version = config["version"]
-    client_json["metadata"]={"version":version,"time":str(time.localtime)}
+    client_json["metadata"] = {"version": version, "time": datetime.utcnow().isoformat() + "Z"}
```

Comment:
I have already made these changes to the driver_code branch. Using .now instead of utcnow

Reply:
We need utc times to keep track of assets across time-zones. now() gives local time.